console/console-countReset-logging-manual.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
```

Comment:
nit: You don't need `<html>` `<head>` or `<body>`.

Reply:
Ah, right! The reason all of the manual tests under `console/` have these unneeded components is because when I added the first one a while ago I just copied the format of some other manual tests in this repo. If you'd like I can remove these extra bits from this test (+ other manuals in this folder)?